### PR TITLE
Android: Splash screen

### LINKED ISSUE #544: Android: SplashScreen
Hello @codervivek5 I want to work on the splash screen of the app

### PR DESCRIPTION
**Pull request by essy16**
@07jasjeet please look into this Pr

### CODE REVIEW

**Comment by 07jasjeet** [comment]
This is not the recommended way of implementing splash screen @essy16. Take a look at this [article](https://proandroiddev.com/splashscreen-api-android-12-part-1-b2272e0ae98e) for more details. Summing up, import `implementation "androidx.core:core-splashscreen:1.10.0"`, use the function `installSpashScreen()` in main activity before `super.onCreate()` and do other mandatory steps such as changing theme in manifest and setting splashscreen theme in theme.xml

**Comment by essy16** [comment]
> This is not the recommended way of implementing splash screen @essy16. Take a look at this [article](https://proandroiddev.com/splashscreen-api-android-12-part-1-b2272e0ae98e) for more details. Summing up, import `implementation "androidx.core:core-splashscreen:1.10.0"`, use the function `installSpashScreen()` in main activity before `super.onCreate()` and do other mandatory steps such as changing theme in manifest and setting splashscreen theme in theme.xml

Is it a must I add the other texts?

**Comment by essy16** [comment]
@07jasjeet please review the changes in my latest commit

File: Android/app/src/main/java/com/vigbag/android/MainActivity.kt [hunk 1]
```diff
@@ -2,14 +2,19 @@ package com.vigbag.android
...
 import androidx.appcompat.app.AppCompatActivity
+import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import androidx.fragment.app.Fragment
 import com.vigbag.android.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     lateinit var binding:ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        Thread.sleep(200)
```
Comment (07jasjeet) [inline Android/app/src/main/java/com/vigbag/android/MainActivity.kt]: Please remove this

File: Android/app/src/main/java/com/vigbag/android/MainActivity.kt [hunk 2]
```diff
@@ -2,14 +2,19 @@ package com.vigbag.android
...
+        installSplashScreen()
         setContentView(R.layout.activity_main)
+        supportActionBar?.hide()
```
Comment (07jasjeet) [inline Android/app/src/main/java/com/vigbag/android/MainActivity.kt]: Why this?

File: Android/app/src/main/res/values/font_certs.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        </item>
```
Comment (07jasjeet) [inline Android/app/src/main/res/values/font_certs.xml]: What is this? Just use .ttf files for fonts and not this.

**Review by 07jasjeet — CHANGES_REQUESTED** [review]

File: Android/app/src/main/java/com/vigbag/android/MainActivity.kt (hunk 2 above)
Comment (essy16) [reply Android/app/src/main/java/com/vigbag/android/MainActivity.kt]: To remove the VigyBag actionbar at the top

File: Android/app/src/main/res/values/font_certs.xml (hunk 3 above)
Comment (essy16) [reply Android/app/src/main/res/values/font_certs.xml]: How do I do this ...I can't understand

Comment (07jasjeet) [reply Android/app/src/main/res/values/font_certs.xml]: https://developer.android.com/develop/ui/views/text-and-emoji/fonts-in-xml

Comment (essy16) [reply Android/app/src/main/res/values/font_certs.xml]: @07jasjeet I've made new commits

Comment (essy16) [reply Android/app/src/main/res/values/font_certs.xml]: @07jasjeet

File: Android/app/src/main/java/com/vigbag/android/MainActivity.kt (hunk 2 above)
Comment (07jasjeet) [reply Android/app/src/main/java/com/vigbag/android/MainActivity.kt]: This shouldn't be needed as we are already using parent theme NoActionBar. Please investigate.

**Review by 07jasjeet — CHANGES_REQUESTED** [review]

File: Android/app/src/main/res/values/font_certs.xml (hunk 3 above)
Comment (07jasjeet) [reply Android/app/src/main/res/values/font_certs.xml]: @essy16 seems like you removed those extra fonts styles altogether. You can take your time if you want to add those fonts. This link I shared above is as good of a guide as it gets.

Comment (essy16) [reply Android/app/src/main/res/values/font_certs.xml]: @07jasjeet Yes I have removed the extra font but I have learnt and also I  have made the latest commits ...please review the changes

**Comment by essy16** [comment]
@07jasjeet

**Comment by 07jasjeet** [comment]
@essy16 Can you please attach a video recording of the feature please?

**Comment by essy16** [comment]
Of the app?https://github.com/codervivek5/VigyBag/assets/58663564/e43c4d97-da63-4596-b7bc-edcd87aa3afc

**Comment by essy16** [comment]
@07jasjeet

**Comment by 07jasjeet** [comment]
> @07jasjeet



> Of the app?https://github.com/codervivek5/VigyBag/assets/58663564/e43c4d97-da63-4596-b7bc-edcd87aa3afc

yes @essy16

**Comment by essy16** [comment]
> > @07jasjeet
> 
> 
> 
> > Of the app?https://github.com/codervivek5/VigyBag/assets/58663564/e43c4d97-da63-4596-b7bc-edcd87aa3afc
> 
> yes @essy16 

I've provided the link to the video @07jasjeet

**Comment by 07jasjeet** [comment]
Alright, LGTM thanks for the work!

**Pull request merged**